src/schema/comments.ts

```diff
@@ -136,6 +150,12 @@ export const typeDefs = /* GraphQL */ `
     edges: [CommentUpvoteEdge!]!
   }
 
+  type MentionUser {
```

Comment:
Why do you define a new type instead of using the existing user type?

Reply:
Oh, right. We can just use the `User` type.